packages/breadboard/tests/index.node.test.ts

```diff
@@ -0,0 +1,217 @@
+// export default Ajv;
+import { Schema, SchemaDraft, ValidationResult, Validator } from "@cfworker/json-schema";
+import breadboardSchema from "@google-labs/breadboard-schema/breadboard.schema.json" assert { type: "json" };
+import assert from "node:assert";
+import { describe, test } from "node:test";
+import { Board } from "../src/board.js";
+
+test("some test", (t) => {
+  assert.equal(1 + 1, 2);
+});
+
+describe("can create a board", (t) => {
+  const board: Board = new Board();
```

Comment:
This is awesome! But, it looks like the old syntax, right? I think we should only write new tests for the new syntax, since we will hopefully remove the old syntax soon.

Reply:
Will update. Just muscle memory as I haven't had chance to use the new syntax yet.

Reply:
https://github.com/breadboard-ai/breadboard/issues/638